.changeset/rude-meals-invite.md

```diff
@@ -0,0 +1,6 @@
+---
+"mermaid": minor
+"@mermaid-js/docs": patch
```

Comment:
```suggestion
```

Reply:
@sidharthv96  We are almost tripling the no. of supported shapes, so @knsv and I feel it should be a minor.

Reply:
@ashishjain0512 that's not the problem, the suggested change is to remove the `@mermaid-js/docs` package, which is not something we release, so it'll throw an error.